### DISCUSSION TITLE
Which version to choose on Windows.

### DISCUSSION THREAD

**Discussion by sidy3d** [BurntSushi/ripgrep#2791] [Q&A]
Hi,

I notice for Windows that there are two releases, one compiled using the MSVC toolchain and another GNU toolchain. What's the recommended version to use? Is there much of a difference between these two when it comes to performance?

Thanks,

**Comment by BurntSushi** [comment]
I would go MSVC personally. That's the Windows "native" version.

I'm not too knowledgeable about Windows myself. I don't remember the precise differences between them, and I very likely cargo culted the binaries for the GNU toolchain. It's more of a "it can be done easily without any additional effort, so I did it."

Maybe a Windows expert can chime in and say more.
